src/editor/components/Main.js

```diff
@@ -84,6 +84,14 @@ export default class Main extends Component {
       '.viewer-logo-start-editor-button'
     );
     htmlEditorButton && htmlEditorButton.remove();
+
+    const isStreetMix = window.location.href.includes('streetmix');
```

Comment:
You can be even more specific here. `window.location.hash` but yeah that's the same.

Reply:
If you want the message also when you click on "Load from Streetmix", you can use this:
```js
    function locationHashChanged() {
      const isStreetMix = window.location.hash.includes('streetmix');
      if (isStreetMix) {
        STREET.notify.warningMessage(
          'Hit save if you want to save changes to the scene. Otherwise changes will be lost'
        );
      }
    }

    locationHashChanged();
    window.addEventListener('hashchange', locationHashChanged);
```

Reply:
good idea, thanks!